lib/msal-browser/src/interaction_client/StandardInteractionClient.ts

```diff
@@ -270,23 +270,35 @@ export abstract class StandardInteractionClient extends BaseInteractionClient {
             validatedRequest.account = account;
         }
 
-        // Check for ADAL SSO
+        // Check for ADAL/MSAL v1 SSO
         if (StringUtils.isEmpty(validatedRequest.loginHint)) {
-            // Only check for adal token if no SSO params are being used
+            // Only check for adal/msal token if no SSO params are being used
             const adalIdTokenString = this.browserStorage.getTemporaryCache(PersistentCacheKeys.ADAL_ID_TOKEN);
             if (adalIdTokenString) {
-                const adalIdToken = new IdToken(adalIdTokenString, this.browserCrypto);
+                this.logger.verbose("Cached ADAL id token retrieved.");
+            }
+
+            // Check for cached MSAL v1 id token
+            const msalIdTokenString = this.browserStorage.getTemporaryCache(PersistentCacheKeys.ID_TOKEN, true);
+            if (msalIdTokenString) {
+                this.logger.verbose("Cached MSAL.js v1 id token retrieved");
+            }
+
+            const cachedIdTokenString = msalIdTokenString || adalIdTokenString;
+            if (cachedIdTokenString) {
+                const cachedIdToken = new IdToken(cachedIdTokenString, this.browserCrypto);
                 this.browserStorage.removeItem(PersistentCacheKeys.ADAL_ID_TOKEN);
```

Comment:
Maybe only do this if you enter the conditional on line 277 just above.  Possibly the same for line 291 just below.

Reply:
Yeah, good idea.